src/vehicle.cpp

```diff
@@ -2881,6 +2881,11 @@ units::mass vehicle::total_mass() const
     return mass_cache;
 }
 
+float vehicle::nowheel_mass() const
```

Comment:
This should return something like `units::quantity<float, units::mass_in_gram_tag>`, same for `wheel_mass`.

You may also convert all the velocity values to something like `quantity<float, velocity_in_meter_per_second_tag>` (or similar, matching the actually used unit). Currently one can not easily see the units involved with all those values and one can easily use those values in the wrong context.